lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1016,10 +1017,10 @@ export class UserAgentApplication {
    * @hidden
    */
   private isInteractionRequired(errorString: string) : boolean {
-    if (errorString.indexOf("interaction_required") !== -1 ||
-    errorString.indexOf("consent_required") !== -1 ||
-    errorString.indexOf("login_required") !== -1) {
-      return true;
+
+    const errorTypes = [InteractionErrorType.INTERACTION, InteractionErrorType.CONSENT, InteractionErrorType.LOGIN];
+    if (errorString && errorTypes.indexOf(errorString) > -1) {
+        return true;
     }
     return false;
```

Comment:
Can be simplified to `return errorString && errorTypes.indexOf(errorString) > -1`

Reply:
Done.